### DISCUSSION TITLE
Improve versioning releases

### DISCUSSION THREAD

**Discussion by Vascom** [OpenHantek/OpenHantek6022#356] [Ideas]
Is it possible instead of "unstable Pre-release" make release with changed fourth number?
Or make release after some finite time (less then 1-2 weeks) from pre-release.

**Comment by Ho-Ro** [comment]
I will investigate what can be done. `unstable` is not meant as a prerelease, it is just a kind of snapshot - more active projects make it regularly and call it `nightly`. 
I could provide intermediate versioned pre-releases, e.g. currently we are at `3.3.2.1-30-ga08dc4b`, but this would clutter the GitHub releases unless I delete the old ones manually.
But I think it will be soon time to release a next version, probably `3.3.2.2` when I'm sure that the HiDPI screenshot didn't introduce regressions - please double check.

**Comment by Vascom** [comment]
I don't want to use a "nightly" numbering scheme for Fedora packages, even for Rawhide. Since it is more convenient when the versions are the same and do not depend on the release of the Fedora, there is less chance of making a mistake.

**Comment by Ho-Ro** [comment]
> Is it possible instead of "unstable Pre-release" make release with changed fourth number?

The `unstable` is just a key press on my system while a numbered release takes some extra manual effort (build a package on a RasPi and upload it, build a FreeBSD package on a virtual FreeBSD and upload it). But as I wrote when the HiDPI issue is stable (please have a look also at your system) I'll roll a new 3.3.2.2 release.
